test/conformance/device_code/CMakeLists.txt

```diff
@@ -10,6 +10,10 @@ else()
     set(AMD_ARCH "${UR_CONFORMANCE_AMD_ARCH}")
 endif()
 
+cmake_path(GET UR_DPCXX EXTENSION EXE)
```

Comment:
Looks like [cmake_path]() was added in CMake 3.20 but we need to support CMake 3.14 for now. Depends what llvm-project or intel/llvm's minimum requirements are actually, if those are above 3.14 then we should also update the `project()` command in the root CMakeLists.txt.

Reply:
Addressed in #1583